ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightsAnalytics.kt

```diff
@@ -0,0 +1,8 @@
+package com.braintreepayments.api
+
+sealed class ShopperInsightsAnalytics(val event: String) {
```

Comment:
I think we're moving away from a sealed class and having analytics strings as string constants within an `object`.

Recent PR with this change: https://github.com/braintree/braintree_android/pull/867/files#diff-fa9e940f487fd9297512740b520d3e1936bb71c7ca35f1ae96a20c3f0316501e